cartography/util.py

```diff
@@ -116,7 +118,11 @@ def timed(*args, **kwargs):
 
 
 # TODO Move this to cartography.intel.aws.util.common
-def aws_handle_regions(func):
+def aws_handle_regions(func=None, default_return_value=[]) -> Callable:
+    """
+    A decorator for returning a default. Use on functions that would return a client error
```

Comment:
Good that we're adding comments here but can we make this clearer -

E.g. what default does it return?
What function(s) is this meant to be used on?

Reply:
Added a better description.